pages/admin/my-jobs.vue

```diff
@@ -46,7 +46,7 @@ export default {
   methods: {
     async getJobs() {
       this.loadingJobs = true
-      const jobs = await this.$jobRepository.index()
+      const jobs = await this.$companyRepository.getJobsByCompany()
```

Comment:
Haven't it  pagination to this list?

Reply:
I can do these changes to use pagination in another MR, after do this using Vuex strategy. What do you think about it?